libraries/Ticker/Ticker.cpp

```diff
@@ -63,25 +60,19 @@ void Ticker::detach()
 		return;
 
 	os_timer_disarm(_timer);
-	delete _timer;
+	//_timer->~ETSTimer();
 	_timer = nullptr;
 	_callback_function = nullptr;
 }
 
 bool Ticker::active() const
 {
-	return (bool)_timer;
+	return _timer;
 }
 
 void Ticker::_static_callback(void* arg)
 {
-	Ticker* _this = (Ticker*)arg;
-	if (_this == nullptr)
-	{
-		return;
-	}
-	if (_this->_callback_function)
-	{
-		_this->_callback_function();
-	}
+	Ticker* _this = reinterpret_cast<Ticker*>(arg);
+	if (!_this) return;
```

Comment:
return on its own line please (no single line ifs). Similar below for callback function.